cpp/src/io/parquet/page_hdr.cu

```diff
@@ -411,6 +411,7 @@ extern "C" __global__ void __launch_bounds__(128)
           bs->cur = bs->end;
         }
       }
+      SYNCWARP();
```

Comment:
`SHFL0` would do the job, but tool gives error, so I am not sure whether to let `SYNCWARP` be used or not.